src/css/Properties.js

```diff
@@ -172,6 +174,8 @@ var Properties = module.exports = {
     "cue-after"                         : 1,
     "cue-before"                        : 1,
     "cursor"                            : 1,
+    "cx"                                : "<coordinate>",
```

Comment:
The coordinate would need to be added to ValidationTypes.js . I'll see if I can do that. You're of course also welcome to add it, and a test.

Reply:
Thanks @frvge,

MDN states the `<coordinate>` content type's syntax is the same as `<length>`. I went ahead and added it to be explicit, but could have simply switched out `<coordinate>` for `<length>` in *Properties.js*. Let me know if you have a preference. Otherwise, I added a test for the `<coordinate>` type.

Reply:
LGTM. Thanks.